cirq/ops/raw_types.py

```diff
@@ -206,6 +238,13 @@ def wrap_in_linear_combination(
         from cirq.ops import linear_combinations
         return linear_combinations.LinearCombinationOfGates({self: coefficient})
 
+    def add_tag(self, tag: Tag):
```

Comment:
consider accepting bare strings and taking care of wrapping in `Tag`

Reply:
Done.